backend/apps/user/src/config/configuration.ts

```diff
@@ -0,0 +1,18 @@
+import { DatabaseConfigurationOptions } from '@app/sql-database';
+
+const userConfiguration = () => {
+  const databaseConfigurationOptions: DatabaseConfigurationOptions = {
+    host: process.env.USER_SERVICE_SQL_DATABASE_HOST,
+    port: process.env.USER_SERVICE_SQL_DATABASE_PORT,
+    user: process.env.USER_SERVICE_SQL_DATABASE_USER,
+    password: process.env.USER_SERVICE_SQL_DATABASE_PASSWORD,
+    database: process.env.USER_SERVICE_SQL_DATABASE_NAME,
+  };
```

Comment:
Can we similarly have a library function that abstracts this database config away? e.g. `getSqlDatabaseConfigurationForService`